Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift

```diff
@@ -75,6 +77,19 @@ public final class CardPresentPaymentStore: Store {
 // MARK: - Services
 //
 private extension CardPresentPaymentStore {
+    func checkOnboardingState(siteID: Int64, onCompletion: (CardPresentPaymentOnboardingState) -> Void) {
```

Comment:
Would it be possible to add some tests for this method?

Reply:
Created an issue for that in #4630